index.html

```diff
@@ -38,6 +39,131 @@ <h1 class="primary-text">I’m Richmond Glad to see you!</h1>
             <li class="angelistic"><a href="#"><img src="./logos/angelist.png" alt="github-logo" /></a>
             </li>
         </ul>
+      </section>
+      <section class="works-container">
+        <div class="projects-cards">
+          <div class="card">
+          <div class="img"><img src="./logos/t2.png" alt="work example" /></div>
+            <p class="card-title">Tonic</p>
+            <div class="tag">
```

Comment:
- [ ] The project title (e.g., "Tonic") should use a header tag `hx`(`<h2>`, `<h3>` etc) instead of `p` tag.